src/commands/push.yaml

```
@@ -0,0 +1,12 @@
+description:
+  Push the current image to your repository
+parameters:
+  version:
```

This will probably break something, so beware.

```suggestion
  image-name:
```

---

I'll change this the after we get the "go ahead" from Kyle, it'll break the build & tests

---

Never mind, fixed